ReactCommon/cxxreact/Value.h

```diff
@@ -13,6 +13,7 @@
 #include <JavaScriptCore/JSValueRef.h>
 
 #include <folly/dynamic.h>
+#include <jni/LocalString.h>
```

Comment:
This is in our ReactCommon package which is also used on iOS (or at least being tested there). As such, we're unable to depend on jni utilities like this. Unfortunately, the best option is to extract this such that it can be shared between the jni/common code. This doesn't seem super trivial to me unfortunately.

I'll post about this internally.